templates/workspaces/base/terraform/appserviceplan.tf

```diff
@@ -0,0 +1,14 @@
+resource "azurerm_app_service_plan" "workspace" {
+  count = var.deploy_app_service_plan ? 1 : 0
```

Comment:
Just a thought, isn't this the first time we might not have the resource we expect to?
I mean, if we try to deploy guacamole, it depends on the workspace being ready for it...

Reply:
Yes, do ive defaulted it to true and not exposed it., so will always be there.

 Down the line after the shared service work we may look at how can update the workspace and enable the app service plan via an output of the workspace service.